### DISCUSSION TITLE
en/lc/2622/

### DISCUSSION THREAD

**Discussion by giscus** [doocs/leetcode#2835] [Announcements]
# en/lc/2622/

LeetCode solutions in any programming language

https://leetcode.doocs.org/en/lc/2622/

**Comment by rain84** [comment]
Hey!

>Once the duration has elapsed, the key should be inaccessible.

There will be no records about expired data, if `.removeExpire()` will run

> `private now(): number {
        return new Date().getTime();
    }`
    
IMHO `Date.now()` is more convinient for such purposes )

**Reply by yanglbme** [reply]
Hello, could you please submit a pull request to improve the existing solution?

**Reply by rain84** [reply]
Shure!

**Reply by rain84** [reply]
@yanglbme done )
